bin/fin

```diff
@@ -3417,18 +3558,34 @@ case "$1" in
 		shift
 		restart
 		;;
-	ps)
-		_ps
+	project)
+		shift
+		case "$1" in
+			status)
+				project_status
+				;;
+			list)
+				shift
+				project_list "$@"
+				;;
+			*)
+				echo -e "Unknown command $1. See ${yellow}fin help project${NC}"
+				exit 1
+				;;
+		esac
+		;;
+	status|ps)
+		project_status
 		;;
-	projects|pr)
+	projects|pl)
```

Comment:
Add a TODO note that this should be removed and "project list" should be used instead.
I still like the shortcut "pl" though, so let's keep that at the top level.